### PR TITLE
chore: Set Windows SIG image verions to fake values

### PR DESCRIPTION
**Pull request by AbelHu**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Set Windows SIG image verions to fake values

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Review by ShiqianTao — APPROVED** [review]

**Review by junjiezhang1997 — APPROVED** [review]

**Review by alexeldeib — APPROVED** [review]
lgtm assuming rp e2e failures are unrelated

would be nice to make it even more obviously fake -- to a new dev, it's very confusing that these images wouldn't work right. maybe "SHOULD_NEVER_BE_USED" or something similarly obvious

but I don't know if we rely on the images being reasonable on RP side

**Comment by AbelHu** [comment]
> lgtm assuming rp e2e failures are unrelated
> 
> would be nice to make it even more obviously fake -- to a new dev, it's very confusing that these images wouldn't work right. maybe "SHOULD_NEVER_BE_USED" or something similarly obvious
> 
> but I don't know if we rely on the images being reasonable on RP side

I think that the e2e failure is not related to this change. We have a wiki about how to update the sig image versions in AgentBaker and will update it again after merging this PR

**Pull request merged**
